dev/io.openliberty.http.monitor_fat/publish/servers/ContainerJustServer/jvm.options

```diff
@@ -1,6 +0,0 @@
--Dcom.ibm.ws.beta.edition=true
--Dotel.metrics.exporter=otlp
--Dotel.traces.exporter=none
--Dotel.logs.exporter=none
--Dotel.sdk.disabled=false
```

Comment:
If you delete this file, where do you set `otel.sdk.disabled=false`?

Reply:
Oops, yes this server would require the jvm.options.